Sindri-Labs/sindri-resources
# Patch mainnet circuit (halo2)

Due to a subdependency conflict (bumpalo), we push an update to our mainnet header circuit that forces an older version of `ethers-providers` and `ethers-core` that avoids the picky crate.

## Testing Instructions:
Submit both of the following to the frontend circuit upload to ensure they were re-compressed and patched properly:
- [x] `circuit_database/halo2/axiom-v0.3.0/mainnet_header.tar.gz`
- [x] `circuit_database/halo2/axiom-v0.3.0/mainnet_header.zip`

I've already shown that the "raw" circuit project uploads successfully via the CLI:
![image](https://github.com/Sindri-Labs/sindri-resources/assets/62887234/45c867e6-f7e3-440f-a1e2-4c8561e14af9)
